server/test/integration/com/thoughtworks/go/server/materials/MaterialDatabaseDependencyUpdaterTest.java

```diff
@@ -402,13 +275,13 @@ private Stage stage(int pipelineCounter) {
         return stage;
     }
 
-    private void stubStageServiceGetHistory(DependencyMaterial dependencyMaterial, Stages... stageses) {
+    private void stubStageServiceGetHistory(DependencyMaterial dependencyMaterial, Stages... stages) {
```

Comment:
Since it's a list of `Stages` objects (already plural), whoever wrote it meant to use `stageses`. :) Not that I'm against changing it.